website/documentation/050-tutorials/content/howto/secure-seccomp/_index.md

```diff
@@ -0,0 +1,985 @@
+---
+title: Custom Seccomp profile
+description: "Custom Seccomp profile"
+type: tutorial-page
+level: advanced
+index: 5
+category: Security
+scope: operator
+aliases: ["readmore/seccomp"]
+---
+
+# Custom Seccomp profile
+
+## Context
+
+[Seccomp](https://en.wikipedia.org/wiki/Seccomp) (secure computing mode) is a security facility in the Linux kernel for restricting the set of system calls applications can make.
+
+Currently in Kubernetes the Seccomp feature and support is `Alpha`. To configure it on a `Pod`, the following annotations can be used:
+
+- `seccomp.security.alpha.kubernetes.io/pod: <seccomp-profile>` where `<seccomp-profile>` is the seccomp profile to apply to all containers in a `Pod`.
+- `container.seccomp.security.alpha.kubernetes.io/<container-name>: <seccomp-profile>` where `<seccomp-profile>` is the seccomp profile to apply to `<container-name>` in a `Pod`.
+
+More details can be found in the `PodSecurityPolicy` [documentation](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#seccomp).
+
+## Installation of custom profile
+
+By default, kubelet loads custom Seccomp profiles from `/var/lib/kubelet/seccomp/`. There are two ways in which Seccomp profiles can be added to a `Node`:
+
+- to be baked in the machine image
+- to be added in runtime.
+
+This guide focuses on creating those profiles via a `DaemonSet`.
+
+Create a file called `seccomp-profile.yaml` with the following content:
+
+```yaml
```

Comment:
I would suggest to use a far simple profile for the purposes of the tutorial because now 80% of the guide is a copy of the docker profile and it would be simpler for users that start just now with the topic:
```json
{
  "defaultAction": "SCMP_ACT_ALLOW",
  "syscalls": [
    {
      "name": "chmod",
      "action": "SCMP_ACT_ERRNO"
    }
  ]
}
```

Source - https://access.redhat.com/documentation/en-us/openshift_container_platform/3.5/html/cluster_administration/admin-guide-seccomp

You could still leave the default docker profile as reference.